query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/NewQueryResource.scala

```diff
@@ -223,7 +233,7 @@ class NewQueryResource(
                 case Right(value) => value
                 case Left(err) => throw new Exception("Invalid error response: " + err.english)
               }
-              Status(other) ~> JsonResp(error)
+              base~> JsonResp(error)
```

Comment:
```suggestion
              base ~> JsonResp(error)
```
I realize this doesn't matter. 😛